src/game/client/draw/w3dmodeldraw.h

```diff
@@ -183,16 +193,27 @@ struct ModelConditionInfo
     uint64_t m_transition;
     float m_animationSpeedFactorMin;
     float m_animationSpeedFactorMax;
-    std::map<NameKeyType, PristineBoneInfo> m_boneMap;
-    TurretInfo m_turretInfo[2];
-    std::vector<WeaponBarrelInfo> m_weaponBarrelInfoVec[3];
-    bool m_hasWeaponBone[3];
-    char m_validStuff;
+    mutable std::map<NameKeyType, PristineBoneInfo> m_boneMap;
+    mutable TurretInfo m_turretInfo[2];
+    mutable std::vector<WeaponBarrelInfo> m_weaponBarrelInfoVec[3];
+    mutable bool m_hasWeaponBone[3];
+    mutable char m_validStuff;

```

Comment:
Is this meant to be called "validstuff"? Maybe could be `int8_t m_validateFlags`. Also, is it really 1 byte?

Reply:
The name is confirmed from BFME2. And yes its confirmed to be 1 byte in size.